e2e/app/page/workspaces-page.ts

```diff
@@ -88,8 +88,8 @@ export default class WorkspacesPage extends WorkspaceEditPage {
     await (await editPage.getWorkspaceNameTextbox()).type(workspaceName);
     await (await editPage.getWorkspaceNameTextbox()).tabKey();
 
-    // select Synthetic Data Set 3
-    await editPage.selectDataSet('3');
+    // select the default Synthetic Dataset
```

Comment:
The default will already be selected. If you intend to test the default behavior, I would not touch the selector